### ISSUE TITLE
Blogger - font for post title

### ISSUE THREAD

**Issue by romona** [OpenLiveWriter/OpenLiveWriter#171]
So far the only thing I've seen as an issue, not a big one, but wondering if it can be corrected. The font for a post title on blogger using OLW, defaults to Arial and won't allow any other option. The box for the font is greyed out. It's fine in the area for the blog post itself, but not the title bar.

Very happy and grateful to those of you who are working so hard to make this work for us. Thanks.

**Comment by ScottIsAFool** [comment]
I don't believe this is an issue and is by design: the blog post title's font is handled by your blog's theme. This isn't unique to Blogger, but all blog types. @willduff, correct me if I'm wrong.

**Comment by romona** [comment]
Hi Scott
I've been using the original WLW for years and have been able to use any
font I wanted for the post titles. i set it up in Customize from the
Blogger dashboard. When I then open WLW to create a post, the font for the
title defaults to what I set up in Customize. It doesn't do that in OLW.

mailto <romona@shaw.ca>:romona.king@gmail.com

WEBSITE:  http://www.northbrookdesigns.com

BLOG1: http://northbrookdesigns.blogspot.com

PINTEREST: http://pinterest.com/romonak/

**Comment by nvella** [comment]
Hi @romona.

Are you still encountering this issue? As @ScottIsAFool mentioned above, Windows Live Writer and Open Live Writer have never had the ability to _edit_ the font of a blog title, as this is controlled by your blog theme. However, Open Live Writer should still be able to display the font from your blog title whilst editing if it correctly detects your blog theme. Please see the screenshot below

![image](https://user-images.githubusercontent.com/8158648/62597513-a84e1d00-b929-11e9-994d-81f15b910538.png)

Cheers 👍

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
